lib/index.js

```diff
@@ -37,6 +37,12 @@ module.exports = {
         'testing-library/no-dom-import': ['error', 'angular'],
       },
     },
+    cypress: {
+      plugins: ['testing-library'],
+      rules: {
+        'testing-library/no-debug': 'warn',
+      },
```

Comment:
Notice here that we don't spread `recommendedRules`

Reply:
I guess we should update the comment in Frameworks section within README saying "Note that frameworks configurations enable their specific rules + recommended rules."